neuralprophet/components/future_regressors/neural_nets.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, config, id_list, quantiles, n_forecasts, device, config_trend
                     regressor_net.append(nn.Linear(d_inputs, self.d_hidden_regressors, bias=True))
                     d_inputs = self.d_hidden_regressors
                 # final layer has input size d_inputs and output size equal to no. of forecasts * no. of quantiles
```

Comment:
^ this comment may need updating. 

Q: Is this NN truly applied separately to each observation and each forecast target?

Reply:
It's been applied for each batch layer by layer